OxfordBodleian/Juel-Jensen/BDLaethe33.xml

```diff
@@ -63,13 +67,15 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                                 Text corresponding to Pss 1 and 2 (beginning) is missing. The end of Ps 151 is missing.
```

Comment:
The text corresponding to Ps 1 and the beginning of Ps 2 is missing. The end of Ps 151 is likewise missing.

On a side note, might it be worth making our Bible abbreviation consistent and decide what style we want to follow? (even if a particular publisher may impose a different style on us)

Reply:
Do you have a preference for a style? (I have none)

Reply:
No but we probably need to be consistent, below Psalm is abbreviate with a full stop after the abbreviation = Ps.